topics/dev/tutorials/bioblend-api/slides.html

```diff
@@ -263,18 +336,32 @@
 - `ex1_galaxy_api.ipynb`: launch a workflow using the **Galaxy API**
 - `ex1_bioblend.ipynb`: launch a workflow using **BioBlend**
 
+???
+
+- Now it's a good time to try the Galaxy API and BioBlend!
+- Follow the instructions on the slide to access the tutorials and exercises.
+
 ---
 
 ## BioBlend limitations
 
 - Python-only (but separate *blend4j* and *blend4php* exist)
 - Its methods just deserialize the JSON response
   - No isolation from changes in the Galaxy API
-  - Need to extract the entity id for further processing
+  - Need to extract the entity ID for further processing
 - No explicit modeling of Galaxy entities and their relationships
 - Complex operations still need many function calls
   - Need for higher-level functionality
 
+???
+
+- Although easier to use than the Galaxy API, BioBlend has some limitations.
+- First of all, it's only available for Python.
+- There are alternative libraries for Java and PHP, but they are less complete.
```

Comment:
A weirdness that I noticed when testing with Mozilla TTS: anything after "PHP" is cut out.

Reply:
Yeahhh, I've had this a ton, and @eancelet experienced this too, everything after `Jupyter,` was cut. Removing the `,` fixed it in Moz TTS but, it works fine either way in AWS.

The mozilla tts has a lot of bugs but it's primary purpose isn't producing the final output, just producing *something* so contributors have an idea of what the final video will look like without having to setup AWS credentials and potentially pay money